packages/commerce-sdk-react/src/hooks/useQuery.ts

```diff
@@ -39,6 +39,12 @@ export const useQuery = <Client extends ApiClient, Options extends ApiOptions, D
     }
 ) => {
     const authenticatedMethod = useAuthorizationHeader(hookConfig.method)
+    const copiedOpts = clone(apiOptions)
+    // remove invalid params so they don't get passed into wrapped method
+    // which will result in  "invalid params" warnings from isomorphic library
+    delete copiedOpts.parameters.clientId
+    delete copiedOpts.parameters.currency
+    delete copiedOpts.parameters.locale
```

Comment:
What about the endpoints where these params are valid? For example, `locale` and `currency` are valid query params for getProducts: https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-products?meta=getProducts

Reply:
@joeluong-sfcc Good point. I've made the changes to allow locale and currency for some hooks when they are valid.